maintainers/maintainer-list.nix

```diff
@@ -3704,6 +3704,12 @@
       fingerprint = "4749 0887 CF3B 85A1 6355  C671 78C7 DD40 DF23 FB16";
     }];
   };
+  DPDmancul = {
+    name = "Davide Peressoni";
+    email = "davide.peressoni@tuta.io";
+    matrix = "@dpd-:matrix.org";
+    githubId = 3186857;
+  };
```

Comment:
Welcome to nixpkgs! This should be done in a seperate commit like `maintainers: add DPDmancul`.